### DISCUSSION TITLE
Welcome to Network Survey Discussions!

### DISCUSSION THREAD

**Discussion by christianrowlands** [christianrowlands/android-network-survey#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shortwavesurfer2009** [comment]
I find cellular to be an interesting topic and want to see similar info to cellmapper but FOSS

**Reply by christianrowlands** [reply]
I have been considering creating a central cellular database for Network Survey similar to CellMapper or WiGLE. It would certainly be a lot of work and my biggest concern is data privacy. I want to make sure to anonymize the location data as good as possible to prevent people from giving up their locations, but it turns out that is harder than I thought.

Anyway, if I did create something like that, how would you want to use it? What is the use case that would cause you to push the upload button in Network Survey and/or go to the website?

**Comment by shortwavesurfer2009** [comment]
My main use case was uploading the bands that towers had to sell mapper, but honestly, I don't know that I would worry about creating something like that because that is a ton of work and I am not sure how much benefit would come of it since cellmapper already exists

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by Unacceptium** [comment]
i have 2 question:
the logging files are CSV and/or geopackage
what is geopackage and why i CAN open it with SqliteEditorAndroid? i never ran into a file extension like this...

my phone does not have the wifi throttling function in dev settings. any advice?(android12)

**Reply by christianrowlands** [reply]
Good questions...

GeoPackage is a SQLite file that has some specific specifications for storing location information. The standard is defined here: https://www.geopackage.org/

You can open a GeoPackage file with any SQLite editor, but to view the location column you need to decode it. I went with GeoPackage because tools like QGIS ( https://qgis.org/en/site ) support GeoPackage out of the box, but it ended up that no one really wanted GeoPackage so I added support for CSV. It is a cool standard but did not really take off the way I thought it was, but I am leaving support in the app for the people that still want to use it.

As for Wi-Fi throttling, you have to turn on developer options first, and then disable the throttling in the developer options settings. Here are some instructions for different model phones...

<img width="994" alt="image" src="https://github.com/christianrowlands/android-network-survey/assets/8020686/5f7ec21c-e572-4637-a127-ea572c376f02">

**Reply by Unacceptium** [reply]
thanks but heres a screenshot (translated) i dont have that option anywhere!
![Screenshot_20240325_074912](https://github.com/christianrowlands/android-network-survey/assets/145642963/68e93b7c-a0d8-4449-8b74-268c081026e0)

**Reply by christianrowlands** [reply]
That is strange. Are you running some sort of custom Android ROM or any other mods that might have removed that option? Or maybe the device manufacturer removed the option? This is what mine looks like:

![image](https://github.com/christianrowlands/android-network-survey/assets/8020686/a4e3a9d9-0b63-4b41-a336-adce15af5827)

**Reply by Unacceptium** [reply]
i have a "honor" device, which has "magic ui" which is hiding a lot of functions, like accent color etc etc etc

**Reply by Unacceptium** [reply]
but i can say something better; wifi in settings app is also throttled! for example, my home wifi got connected almost instantly but the settings show the mall's wifi where im been a hour ago. my phone is chineese sadly

edit:and no, not for seconds... sometimes more than 5 mins!

**Reply by christianrowlands** [reply]
Hmm, yeah, I don't think I have any ideas then. It sounds like it is a limitation of the modifications the device manufacturer made to Android. You might be able to load a custom ROM that is closer to the stock Android OS, but that typically requires root and running around with rooted device has other problems such as security vulnerabilities.
